style.css

```diff
@@ -1,3 +1,114 @@
-h1 {
-  color: blue;
+* {
+  --main-bg-color: #fff;
+  --header-text-color: #172b4d;
+  --normal-text-color: #344563;
+  --secondary-text-color: #6070ff;
+  --light-text-color: #7f8cff;
+  --main-font-family: 'Poppins', sans-serif;
+
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Great job using CSS variables to avoid repeating code! 💯

Reply:
### I have applied all the suggested solutions